File: plugins/uk.ac.gda.devices.bssc/embl/App/script/TestScript.py
import time
import org.embl.task.Script
import org.embl.ctrl.Device
import org.embl.dev.tecan.TecanValve
import org.embl.bssc


class TestScript(org.embl.task.Script):
    def run(self, parent, args):
        # @type parent org.embl.hc.hcControl
        ctr=parent
        ctr.setValvePosition(org.embl.bssc.scControl.DEVICE_SYRINGE_VALVE, org.embl.dev.tecan.TecanValve.Position.output)
        return "OK"

File: plugins/uk.ac.gda.devices.bssc/embl/App/script/TestScript.java

import org.embl.bssc.scDevPlateTable.WellPosition;
import org.embl.dev.tecan.TecanValve.Position;

public class TestScript extends org.embl.task.Script
{
    @Override
    protected Object run(Object parent, Object[] args) throws Exception
    {
        org.embl.bssc.scControl ctr = (org.embl.bssc.scControl) parent;
        ctr.setValvePosition(org.embl.bssc.scControl.DEVICE_SYRINGE_VALVE, Position.output);
        ctr.log(this, "Done");
        ctr.movePlateWell(0, 0, 0, WellPosition.TOP);
        Thread.sleep(1000);
        ctr.movePlateWell(0, 0, 0, WellPosition.BOTTOM);
        Thread.sleep(1000);
        ctr.movePlateWell(0, 0, 0, WellPosition.TOP);
        Thread.sleep(1000);
        ctr.movePlateWell(0, 0, 0, WellPosition.BOTTOM);        
        return true;
    }
}